src/index.html

```diff
@@ -5,9 +5,45 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body data-qa="card">
+    <section class="card">
+      <div class="product_image">
+        <img src="images/imac.jpeg" alt="iMak">
+      </div>
+
+      <h1 class="product_title">
+        APPLE A1419 iMac 27" Retina
+        <br>
+        5K Monoblock (MNED2UA/A)
+      </h1>
+
+      <h2 class="product_code">Product code: 195434</h2>
+
+      <div class="stars">
+        <section class="stars__section">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </section>
+
+        <h2 class="stars__title">Reviews: 5</h2>
+
+      </div>
+
+      <div class="price">
+        <h2 class="price__title">Price:</h2>
+        <h3 class="price__number">$2,199</h3>
+      </div>
+      <div class="button" data-qa="hover">
+        <a href="#" class="button__buy" >BUY</a>
+      </div>
```

Comment:
```suggestion
      <div class="button" data-qa="hover">
        <a href="#" class="button__buy" >BUY</a>
      </div>
```

You don`t need this additional container, delete div and merge attributes and styles.